public/app/features/manage-dashboards/components/ImportDashboardForm.tsx

```diff
@@ -136,6 +147,18 @@ export const ImportDashboardForm: FC<Props> = ({
             </Field>
           );
         })}
+      <ImportDashboardLibraryPanelsList
+        inputs={newLibraryPanels}
+        label="New library panels"
+        description="Lists all new library panels. These library panels will be imported."
```

Comment:
@achatterjee-grafana I could use an extra pair of eyes on these two texts, thank you.